bip-0021.mediawiki

```diff
@@ -11,6 +11,12 @@
   Created: 2012-01-29
 </pre>
 
+=Superseded by BIP 321=
+
+This BIP has been superseded and replaced with BIP 321. Please see [[bip-0321.mediawiki|BIP 321]] instead.
+
+=Original BIP=
+
```

Comment:
This PR is just adding BIP 321 as Draft, so it might be premature to mark BIP 21 as superseded. It would be more fitting when BIP 321 is moved from Proposed to Final. 

Per the BIP 2 process, when BIP 21 is marked as superseded, please use the `Superseded-By: <BIP number>` header in the preamble for that purpose and update the status of BIP 21 to "Replaced".

Reply:
As far as I am aware, it hasn’t taken that long in the past year, as long as a reasonable argument was made why something should be final.